src/components/backend-ai-settings-view.ts

```diff
@@ -301,23 +314,17 @@ export default class BackendAiSettingsView extends BackendAIPage {
                   <div class="horizontal layout setting-item">
                     <div class="vertical center-justified layout setting-desc-select" style="margin: 15px 0px;">
                       <div class="title">${_t('settings.Scheduler')}</div>
-                      <div class="description-shrink">${_t('settings.JobScheduler')}<br/>
+                      <div class="description-shrink">${_t('settings.SchedulerConfiguration')}<br/>
                           ${_t('settings.Require1912orAbove')}
                       </div>
                     </div>
-                    <div class="vertical layout center-justified">
-                      <mwc-select id="scheduler-switch"
-                                  required
-                                  outlined
-                                  @selected="${(e) => this.changeScheduler(e)}"
-                                  label=""
-                                  style="width:130px;">
-                        ${this.jobschedulerType.map((item) => html`
-                          <mwc-list-item value="${item}"
-                                        ?selected=${this.options['scheduler'] === item}>
-                            ${item}
-                          </mwc-list-item>`)}
-                      </mwc-select>
+                    <div class="vertical center-justified layout">
+                      <mwc-button
```

Comment:
How about adding some padding in the button? For now, It looks too densed.
![스크린샷 2021-05-10 오후 1 28 08](https://user-images.githubusercontent.com/46954439/117606358-aebf9780-b194-11eb-982a-de1ccc932d15.png)

Reply:
Oops, my language configuration is Korean so I didn't know that. I'll add some padding.